examples/ensemble/plot_ensemble_oob.py

```diff
@@ -0,0 +1,86 @@
+"""
+=============================
+OOB Errors for Random Forests
+=============================
+
+The ``RandomForestClassifier`` is trained using *bootstrap aggregation*. During
+training, each new tree is fit from a bootstrap sample of the training
+observations :math:`z_i = (x_i, y_i)`. The *out-of-bag* (OOB) error is the
+average prediction error for each :math:`z_i` from trees that do not contain
+:math:`z_i` in their respective bootstrap sample. This allows models to be
+simultaneously fit and validated [1].
```

Comment:
Thanks for the reference. I guess the cross-validated is more clear in their text as they refer to the exercise that shows the identity. I think you text now is good.